Update Windows and RBAC sections in Logging (#2869)
rancher/docs: content/rancher/v2.x/en/logging/v2.5/_index.md

- [Configuring the Logging Application](#configuring-the-logging-application)
- [Working with Taints and Tolerations](#working-with-taints-and-tolerations)

# Changes in Rancher v2.5

The following changes were introduced to logging in Rancher v2.5:

> - The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers Rancher's logging solution in place of the former, in-house one.

Reviewer: ```suggestion
- The [Banzai Cloud Logging operator](https://banzaicloud.com/docs/one-eye/logging-operator/) now powers Rancher's logging solution in place of the former, in-house solution.
```

Author: done

    - "devteam-splunk"
```

**Unsupported Ouput:** For the final example, we create an output to write logs to a destination that is not supported out of the box (e.g. syslog):

> 

Reviewer 2: now that we support syslog we should update this example to use the next most popular unsupported output.

Reviewer 2: That could be a different ticket though.

Author: That would probably be best. I believe this example uses a different syslog plugin, so it *should* be fine for the immediate future. I do agree though that we should evaluate the next most popular one for a future PR/issue.

# Working with Taints and Tolerations

"Tainting" a Kubernetes node causes pods to repel running on that node.
Unless the pods have a `toleration` for that node's taint, they will run on other nodes in the cluster.
[Taints and tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) can work in conjunction with the `nodeSelector` [field](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) within the `PodSpec`, which enables the *opposite* effect of a taint.
Using `nodeSelector` gives pods an affinity towards certain nodes.
Both provide choice for the what node(s) the pod will run on.

### Default Implementation in Rancher's Logging Stack

By default, Rancher taints all Linux nodes with `cattle.io/os=linux`, and does not taint Windows nodes.
The logging stack pods have `tolerations` for this taint, which enables them to run on Linux nodes.
Moreover, we can populate the `nodeSelector` to ensure that our pods *only* run on Linux nodes.
Let's look at an example pod YAML file with these settings...

```yaml
apiVersion: v1
kind: Pod

> # insert metadata

Reviewer: nit: 
```suggestion
# metadata...
```
same for containers, thats the way I've seen it done in other documentation. We should match whatever K8s docs do.

    kubernetes.io/os: linux
```

In the above example, we ensure that our pod only runs on Linux nodes, and we add a `toleration` for the taint we have on all of our Linux nodes.
You can do the same with Rancher's existing taints, or with your own custom ones.

### Windows Support

> Clusters with Windows workers support logging, but Windows node logs are currently unable to be exported.

Reviewer: ```suggestion
Clusters with Windows workers support exporting logs from Linux nodes, but Windows node logs are currently unable to be exported.
```
